webpack/scenes/ContentViews/Details/Versions/ContentViewVersions.js

```diff
@@ -113,6 +133,30 @@ const ContentViewVersions = ({ cvId }) => {
     }
   }, [response, setMetadata, buildActiveTaskCells, buildCells, dispatch, loading, setRows]);
 
+  const onPromote = ({ cvVersionId, cvVersionName, cvVersionEnvironments }) => {
+    setVersionIdToPromote(cvVersionId);
+    setVersionNameToPromote(cvVersionName);
+    setVersionEnvironments(cvVersionEnvironments);
+    setPromoting(true);
+  };
+
+  const actionResolver = () => [
+    {
+      title: __('Promote'),
+      onClick: (_event, rowId, rowInfo) => {
```

Comment:
do we need to check on anything here before promoting?

Reply:
The answer is no, as we don't have the API's data prior to hitting this button to know if it is disabled or not.